### PR TITLE
Allow usage of important directive

### LINKED ISSUE #6: The process does not seem to keep the ''!important'' directive.
source : 
```css
.tw-h-screen {
    height: 100vh !important;
}
```

generated: 
```css
.tw-h-screen {
    height: 100vh !important;
    height: calc(1vh * 100);
    height: calc(var(--vh, 1vh) * 100);
}
```

### PR DESCRIPTION
**Pull request by menthol**
Fix #6

### CODE REVIEW

**Comment by Faisal-Manzer** [comment]
How many directives are there?

File: index.js [hunk 1]
```diff
@@ -12,13 +12,18 @@ module.exports =
...
+          let important = decl.important;
 
           // Checking that declaration has `vh` and is not corrected.
           let isMatch = value.match(finderRegex) !== null;
           let isPreParsed = value.match(excludeRegex) === null;
           if (isMatch && isPreParsed) {
             let correctedViewport = value.replace(finderRegex, replaceBy);
 
+            if (important) {
+              correctedViewport += ' !important';
```
Comment (Faisal-Manzer) [inline index.js]: Is there any other way to get this directive working? i.e. You used `decl.important` to get isImportant but then used string to specify directive.

Comment (menthol) [reply index.js]: We can use the raw value instead

Comment (Faisal-Manzer) [reply index.js]: How? can I get some link or code changes.

Comment (menthol) [reply index.js]: We get this structure : 
```json 
{
  "raws": {
    "before": " ",
    "between": ": ",
    "important": " ! important"
  },
  "type": "decl",
  "source": {
    "start": {
      "line": 1,
      "column": 29
    },
    "input": {
      "css": "a{ height: 10vh !important; max-height: 20vh ! important; color: #FF10GG; }",
      "hasBOM": false,
      "id": "<input css 7>"
    },
    "end": {
      "line": 1,
      "column": 57
    }
  },
  "prop": "max-height",
  "important": true,
  "value": "20vh"
}

```

Comment (menthol) [reply index.js:24]: Do you want an updated PR with the raw important property ?

Comment (menthol) [reply index.js]: In fact, if the important directive is ` !important` (spaces are important here),  the raws.important property do not exists :
```json
{
  "raws": {
    "before": " ",
    "between": ": "
  },
  "type": "decl",
  "source": {
    "start": {
      "line": 1,
      "column": 4
    },
    "input": {
      "css": "a{ height: 10vh !important; max-height: 20vh ! important; color: #FF10GG; }",
      "hasBOM": false,
      "id": "<input css 7>"
    },
    "end": {
      "line": 1,
      "column": 27
    }
  },
  "prop": "height",
  "important": true,
  "value": "10vh"
}

```

Comment (menthol) [reply index.js]: So we can use  `decl.raws.important || ' !important'`

Comment (menthol) [reply index.js]: In fact, post css use this trick too: 
https://github.com/postcss/postcss/blob/21ce39c7aa19f1b0b237a5a180a59bf786111fdf/lib/stringifier.es6#L45

Comment (menthol) [reply index.js]: Force pushed to use the same way than postcss

**Review by Faisal-Manzer — APPROVED** [review]

**Pull request merged**